### ISSUE TITLE
mergeGeoJson.R Date Error

### ISSUE THREAD

**Issue by MicheleTobias** [UCDavisLibrary/ava#926]
When I try to run the `mergeGeoJson.R` file to create the aggregated files, it's throwing an error on line 35. It's having an issue with the date formats. I suspect this is due to a formatting issue with the dates in the newer boundaries.  Can either @alisonsnwong or @erklopez  take a look at the code and the input data and see if you can track down the problem?

**Comment by MicheleTobias** [comment]
New boundaries:

- high_valley
- long_valley-lake_county
- north_coast

**Comment by MicheleTobias** [comment]
Remember that our attribute standards are here: https://ucdavislibrary.github.io/ava/data.html

**Comment by alisonsnwong** [comment]
I'll try working on it!

**Comment by alisonsnwong** [comment]
I found out there were date format errors in winter_highlands and long_valley-lake_county that threw errors in line 35. I fixed those in QGIS and added a parameter "%m/%d/%y" in the as.Date() functions to keep the format consistent. There was also an error in line 35 that said the long_valley-lake_county wasn't valid. I cross-checked it in QGIS and it was fine. This sf_use_s2(FALSE) solves the duplicated vertex error.
